extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -23,6 +23,26 @@ import {toggle} from '../../../src/style';
 
 
 class AmpStickyAd extends AMP.BaseElement {
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {!Element} */
```

Comment:
Ha! I see what @cramforce means now. This is the perfect solution I am looking for! fixed. PTAL